ui/app/templates/components/clients/current.hbs

```diff
@@ -19,7 +19,16 @@
         FILTERS
         <Toolbar>
           <ToolbarFilters>
-            {{! ARG TODO more filters for namespace here }}
+            <SearchSelect
+              @id="namespace-search-select-monthly"
+              @options={{this.namespaceArray}}
+              @selectLimit="1"
+              @disallowNewItems={{true}}
+              @fallbackComponent="input-search"
+              @onChange={{this.selectNamespace}}
+              @placeholder={{"Filter by namespace"}}
+              @displayInherit={{true}}
```

Comment:
because I needed the styling on the component itself I had to do a class binding. Not my favorite but couldn't find another way to solve for this.